dig.go

```diff
@@ -111,6 +111,25 @@ type InvokeOption interface {
 	unimplemented()
 }
 
+// A dotNode represents a node in a Container's DOT-format graph.
```

Comment:
nit: Given that `dotGraph` includes the context with respect to the `DOT-format` graph, we could make this more focused towards how it relates to the other data structures and remove the `DOT-format` note.

Something like:
```
// A dotNode represents a single node in a dotGraph.
```